app/assets/styles/main.css

```diff
@@ -115,7 +115,7 @@ margin: 0 auto;
 }
 
 .nav-bar {
-  background-color: #1F9DBF;
+  background-color: darkgrey;
```

Comment:
I was testing things out w/ the blend mode on the fly, but I definitely want to switch over to a hex color that's in the palette!